src/model_DAforcing.py

```diff
@@ -131,6 +132,17 @@ def __init__(self, bmi_cfg_file=None):
 
             # read in metadata for BMI compliant arrays:
 
+            if lastobs_file:
```

Comment:
maybe check if `self._lastobs_df` is not empty here instead of if the file name exists? Not sure if an empty dataframe would cause `_bmi_disassemble_lastObs` trouble.